src/widgets/SimpleTable.js

```diff
@@ -0,0 +1,187 @@
+/* eslint-disable react/no-unused-prop-types */
```

Comment:
I use all the props, I just pass them to a method (`localStyles`) which is not in the class. I can see how this is different from normal use and can revert all the styles if needed ( I just like this as it makes the component so much clearer without having all the styles logic ).

Reply:
i don't get this linter error in vscode, does it come up when you try to commit ?

Reply:
![image](https://user-images.githubusercontent.com/35858975/56328967-804d7900-61d5-11e9-9c17-f286236f7e96.png)


Because the props aren't used in  the method

Reply:
d07b0e478c4810b25deee727f559ecf2ec2d4fc6